spark.Dockerfile

```diff
@@ -5,6 +5,9 @@ USER root
 ENV PYTHONPATH=/spf
 ENV CLASSPATH=/spf/jars:$CLASSPATH
 
+# remove the older version of entrypoints with apt-get because that is how it was installed
+RUN apt-get remove python3-entrypoints -y
```

Comment:
need to remove python3-entrypoints because mlflow uses a newer version and this is installed as part of the base image and cannot be uninstalled by pip because it is a system level package.